firestore/scripts/FriendlyEats.View.js

```diff
@@ -312,12 +342,33 @@ FriendlyEats.prototype.viewRestaurant = function(id) {
   return this.getRestaurant(id)
     .then(function(doc) {
       var data = doc.data();
+      if (that.userFavorites.indexOf(id) >= 0) {
+        console.log('This is a favorite restaurant');
+        data['isFavorite'] = true;
+      }
+
+      // This seems really old-school. Wonder if there's a better way
+      var urlParams = new URLSearchParams(window.location.search);
+      if (urlParams.get('fromFavorites')) {
+        console.log('You visited here form your favorites location');
+        data['fromFavorites'] = true;
```

Comment:
Same as above:

Since there's no special chars we can use regular dot notation:

```ts
data.fromFavorites = true;
```